general/OKRs/README.md

```diff
@@ -0,0 +1,156 @@
+
+### Vision:
+
+* Being the leader of AI on Code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Company Wide:
+
+* Having successful users of source{d}
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Being a better company to work at
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Product:
+
+* Launch of hosted playground
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Build a hosted MVP for open-source projects of source{d} PR* (ability to add applications that return results in PR's on GitHub)
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Launch of deduplication of code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Decide on the faith of legacy projects
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+\* we're lost on the name still, this objective needs its own document to explain better since it's a new product
+
+### Developer Community:
+
+* Create a network of influencers incl. Developers, Companies, Academia
+  * Create a forum (tbd whether it's slack, mailing list, or others) for influencers.
+  * Identify 20 influencers and invite them to join the community.
+  * Meet with all of the influencers that accept the invitation and welcome them personally.
+* Increase our presence in the developer community in Madrid & SF
+  * Organize 2 Go meetups in Madrid
+  * Co-host 2 Go meetups in San Francisco (Holberton school could host)
+  * Creat a Source Code Analysis meetup in San Francisco and run an inaugural event
+* Host and attend conferences and meetups
+  * Source{d} themed talks at 5 conferences or meetups
+  * Source{d} speakers at 5 large (+200 attendees) conferences
+* Keep track of who you met that was relevant to source{d}/relevant new topics in the MLoSC community
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Publish blogposts at a regular pace
+  * Plan weekly schedule for blogs
+  * Publish 8 blog posts during Q1
+* Start cross posting to Medium
+  * Publish 10 posts to Medium
+  * Once we have enough analytics decide whether it's worth cross posting or simply migrating to medium. 
+* Start publishing videos
+  * Create YouTube channel for source{d}
+  * Create monthly schedule for videos and themes
+  * Write 4 detailed scripts
+  * Publish 2 videos
+* Publish a page where we show all source{d} related videos
+  * Redesign talks page
+  * Include all old videos and the ones we'll publish in the meanwhile
+  * Include 5 talks by non source{d} employees
+* Redefine how we use social media and community Slack
+  * Gather statistics of current usage and views on social media
+  * Write a mission statement for our social media and formalize who has access/responsibilities
+  * Once we have statistics on community size, establish new OKRs on growth
+
+### Academia:
+
+* Getting academia to use our tech
+  * source{d} technologies mentioned in at least one research paper by non-source{d} researcher
```

Comment:
It is very archibable, is not expect te be archive at the 100%, the 70% should be fisable. Maybe 2?

Reply:
Not clear it's that achievable, to be honest.
What do you think @vmarkovtsev?

Reply:
This is very random IMHO. We can put any number we want but the actual one will be the actual one. It's the first time we do this so we have no idea how our efforts will turn out. I like the original binary goal.

Reply:
ok, I'll keep to "being mentioned at least once" in that case